zebra-rpc/src/config.rs

```diff
@@ -30,6 +30,22 @@ pub struct Config {
     /// They can also query your node's state.
     pub listen_addr: Option<SocketAddr>,
 
+    /// IP address and port for the indexer RPC server.
+    ///
+    /// Note: The indexer RPC server is disabled by default.
+    /// To enable the indexer RPC server, compile `zebrad` with the
+    /// `indexer` feature flag and set a listen address in the config:
+    /// ```toml
+    /// [rpc]
+    /// indexer_listen_addr = '127.0.0.1:8232'
```

Comment:
> Should we suggest a different port for the indexer?

Yeah, I think we should, how about `8231`?

```suggestion
    /// indexer_listen_addr = '127.0.0.1:8231'
```